stable/sumologic-fluentd/templates/_helpers.tpl

```diff
@@ -2,22 +2,30 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "name" -}}
+{{- define "sumologic-fluentd.name" -}}
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
 {{- end -}}
 
 {{/*
 Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 */}}
-{{- define "fullname" -}}
+{{- define "sumologic-fluentd.fullname" -}}
 {{- $name := default .Chart.Name .Values.nameOverride -}}
 {{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
 {{- end -}}
 
 {{/*
 A uniquely named secret, which includes the install time
 */}}
-{{- define "fullname-secrets" -}}
-{{- printf "%s-secrets-%d" (include "fullname" .) .Release.Time.Seconds -}}
+{{- define "sumologic-fluentd.fullname-secrets" -}}
+{{- printf "%s-secrets-%d" (include "sumologic-fluentd.fullname" .) .Release.Time.Seconds -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified fluentd user conf name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+*/}}
+{{- define "sumologic-fluentd.fluentdUserConfig.fullname" -}}
```

Comment:
This should be based on the fullname template and just add a suffix. But why do you want a different name at all? I'd just use the standard fullnsme template.

Reply:
> This should be based on the fullname template and just add a suffix

Ahh yes, done https://github.com/kubernetes/charts/pull/3991/commits/2d094963d114c7587f02ed0ee8ae6da7a1a6f622

> But why do you want a different name at all? I'd just use the standard fullnsme template.

I was thinking they might want flexibility to support other configmaps. This one is specific to injecting the fluentd user config